precompiles/Precompiles.sol

```diff
@@ -184,10 +184,9 @@ contract Precompiles {
     /// @notice Deploys all V2 Core contracts in the following order:
     ///
     /// 1. {SablierV2NFTDescriptor}
-    /// 2. {SablierV2LockupDynamic}
-    /// 3. {SablierV2LockupLinear}
-    /// 4. {SablierV2LockupTranched}
-    /// 5. {SablierV2NFTDescriptor}
+    /// 2. {SablierV2LockupLinear}
```

Comment:
Since this is the order of the contracts in which they are deployed. It should be 

```solidity
SablierV2LockupDynamic
SablierV2LockupLinear
SablierV2LockupTranched
```